### PR TITLE
[AdminApiBundle] Added api endpoint for /orders

### PR DESCRIPTION
**Pull request by JaisDK**
| Q               | A
| --------------- | -----
| Branch?         | 1.2
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | 
| License         | MIT

Order API is missing the `/orders` endpoint described in the docs
https://docs.sylius.com/en/1.2/api/orders.html

### CODE REVIEW

**Comment by pjedrzejewski** [comment]
Hi @JaisDK, thanks for your PR! We would need to have some API tests for these endpoints to merge this one. There are some already here: https://github.com/Sylius/Sylius/blob/master/tests/Controller/OrderApiTest.php

**Comment by JaisDK** [comment]
I'll look into it

**Comment by JaisDK** [comment]
@pjedrzejewski I am getting an error "The command "etc/travis/run-suite script "${SYLIUS_SUITE}"" exited with 2." even before adding the new tests. Do you know what is causing this?

**Comment by loevgaard** [comment]
What's the status here, @pjedrzejewski ?

File: src/Sylius/Bundle/AdminApiBundle/Resources/config/routing/order.yml [hunk 1]
```diff
@@ -10,6 +10,15 @@ sylius_admin_api_order:
         serialization_version: $version
     type: sylius.resource_api
 
+sylius_admin_api_orders:
+    resource: |
+        alias: sylius.order
+        section: admin_api
+        only: [index]
```
Comment (lchrusciel) [inline src/Sylius/Bundle/AdminApiBundle/Resources/config/routing/order.yml]: Can you merge this configuration with the configuration above and change `only: [show]` to `only: [show, index]`?

**Comment by lchrusciel** [comment]
The base of this pull-request was changed, you need fetch and reset your local branch 
if you want to add new commits to this pull request. **Reset before you pull, else commits
may become messed-up.**

Unless you added new commits (to this branch) locally that you did not push yet, 
execute `git fetch origin && git reset "add-orders-index-api-endpoint"` to update your local branch.
 
Feel free to ask for assistance when you get stuck :+1:

File: tests/Controller/OrderApiTest.php [hunk 2]
```diff
@@ -17,6 +17,32 @@
 
 final class OrderApiTest extends CheckoutApiTestCase
 {
+
```
[the remark this answers is not in the document]
Comment (JaisDK) [reply tests/Controller/OrderApiTest.php]: which part is redundant? The test class already exists

[the remark this answers is not in the document]
Comment (lchrusciel) [reply tests/Controller/OrderApiTest.php]: Just redundant blank line, nothing important :)

**Comment by Zales0123** [comment]
@JaisDK can you take a look a the failing build? It would be great to have it merged :)

**Comment by JaisDK** [comment]
@Zales0123 The issues doesn't seem to have anything to do with my code changes, så it's a little bit difficult for me to fix.

**Comment by JaisDK** [comment]
@Zales0123 Were you successful?

**Comment by Zales0123** [comment]
Yes, `1.2` is now passing, you can rebase to the newest version to this branch and check out is your build still failing :)

**Comment by JaisDK** [comment]
@Zales0123 New errors :)

**Comment by loevgaard** [comment]
Can we look into this? It would be *great* to have in v1.4! Is there something we can do, @Zales0123?

**Comment by JaisDK** [comment]
1.3 would be great too :)

**Comment by Zales0123** [comment]
Hello, sorry for the response delay. Does this tests also fail on your local machine, @JaisDK?

**Comment by JaisDK** [comment]
#10138

**Pull request closed without merging**